Cannot find module: avif_node_enc_mt.worker-be931951.js [Windows]

**Describe the bug**
I wanted to try running cli on Win10 but I got error:
```
PS C:\Users\konta> squoosh-cli --avif "{speed:5,cqLevel:18,subsample:2}" Q:\test.jpg
0/1 ⠦ ▐▨▨▨▨▨╌╌╌╌╌▌ Encoding (8 threads)
 Q:\test.jpg: 258.25KB
node:internal/event_target:646
  process.nextTick(() => { throw err; });
                           ^
Error: Cannot find module 'C:\C:\Users\konta\AppData\Roaming\npm\node_modules\@squoosh\cli\node_modules\@squoosh\lib\build\avif_node_enc_mt.worker-be931951.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at MessagePort.<anonymous> (node:internal/main/worker_thread:187:24)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:460:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:23:28)
Emitted 'error' event on Worker instance at:
    at Worker.[kOnErrorMessage] (node:internal/worker:298:10)
    at Worker.[kOnMessage] (node:internal/worker:309:37)
    at MessagePort.<anonymous> (node:internal/worker:210:57)
    at MessagePort.[nodejs.internal.kHybridDispatch] (node:internal/event_target:460:20)
    at MessagePort.exports.emitMessage (node:internal/per_context/messageport:23:28) {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
I did try to run `npm i -g @squoosh/cli@latest` and `npx @squoosh/cli`. In both cases the result was the same.

**Version:**
 - OS w/ version: Win10 Pro
 - Node version: v16.6.2
 - npm version: 7.20.5

--- a/libsquoosh/src/emscripten-utils.ts
+++ b/libsquoosh/src/emscripten-utils.ts
@@ -19,7 +19,7 @@ export function instantiateEmscriptenWasm<T extends EmscriptenWasm.Module>(
       // These will have changed in the bundling process and
       // we need to inject them here.
       if (requestPath.endsWith('.wasm')) return pathify(path);
-      if (requestPath.endsWith('.worker.js')) return new URL(workerJS).pathname;
+      if (requestPath.endsWith('.worker.js')) return pathify(workerJS);
       return requestPath;
     },
   });

